test/fixtures/tracks/fake/config.json

```diff
@@ -2,6 +2,7 @@
   "slug": "fake",
   "language": "Fake",
   "repository": "https://github.com/exercism/xfake",
+  "checklist_issue": "1",
```

Comment:
I felt really dirty adding that as a String. Mostly, I was anticipating a conversion error when adding the repo url with the checklist number....but just realized that's what interpolation is for 😅